python-sdk/src/forta_agent/alert_test.py

```diff
@@ -0,0 +1,19 @@
+from .utils import get_alerts;
+
+def test_get_alerts_returns_more_than_one_alert():
+        response = get_alerts({'bot_ids': ["0x79af4d8e0ea9bd28ed971f0c54bcfe2e1ba0e6de39c4f3d35726b15843990a51"], 'created_since': '1668635340'})
+        assert len(response.alerts) > 0
+
+def test_get_alerts_parse_alert():
+        response = get_alerts({'bot_ids': ["0x79af4d8e0ea9bd28ed971f0c54bcfe2e1ba0e6de39c4f3d35726b15843990a51"], 'created_since': '1668635340'})
+        alert = response.alerts[0]
+        print(alert.description)
+        assert True
```

Comment:
(what is this testing?)

Reply:
This is testing that the python class correctly serializes the graphql response. If there is an error the test fails.